Cargo.toml

```diff
@@ -51,7 +51,13 @@ url = "2.2.0"
 hyper = "0.14"
 hyperlocal = "0.8"
 # close to same tokio version as dependent by hyper v0.14 and hyperlocal 0.8 -- things can go awry if they mismatch
-tokio = { version = "1", features = ["sync", "macros"] }
+tokio = { version = "1", features = ["sync", "macros", "rt-multi-thread", "rt"] }
+opentelemetry = { version = "0.20.0", features = ["rt-tokio"] }
```

Comment:
6 new (direct) dependencies, oh my :sweat_smile:

Should we perhaps put this behind a feature flag so that the extra dependencies could be avoided by users that don't require tracing?

Reply:
I added a feature flag to this. By default tracing is off and can be turned on with:
`--no-default-features --features otlp-tracing`